### PR TITLE
QT: Better text colors in the debugger

### PR DESCRIPTION
**Pull request by Mte90**
immagine
immagine

### CODE REVIEW

**Comment by Mte90** [comment]
Before that I continue to work on that is helpful? There is anything that I need to know to avoid code duplication?

I cannot find the definition of this method `debugger->backend->printf` to replicate the behaviour.

File: include/mgba/debugger/debugger.h [hunk 1]
```diff
@@ -99,6 +99,7 @@ struct mDebuggerPlatform {
 	void (*deinit)(struct mDebuggerPlatform*);
 	void (*entered)(struct mDebuggerPlatform*, enum mDebuggerEntryReason, struct mDebuggerEntryInfo*);
 
+	bool (*supportHTML)(struct mDebuggerPlatform*);
```
Comment (Mte90) [inline include/mgba/debugger/debugger.h:102]: Probably I can remove this line

File: src/platform/qt/DebuggerConsole.ui [hunk 2]
```diff
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>480</width>
+    <width>508</width>
```
Comment (Mte90) [inline src/platform/qt/DebuggerConsole.ui:9]: better round to 510

**Comment by endrift** [comment]
I'm not really interested making the backend support HTML just because the frontend can render it. HTML is bad here in a number of ways, and the debugger console is kind of temporary until the real debugger UI is done.

**Comment by Mte90** [comment]
I know that HTML is not very good but it was an easy way to get a better output and maybe also to open new integration to mgba.

**Pull request closed without merging**